### DISCUSSION TITLE
autoPlay

### DISCUSSION THREAD

**Discussion by matheuskousen** [vercel/next.js#17179] [Help] [answered]
autoPlay aren't work , anyone can i help me ?

**Comment by jamesmosier** [comment, marked as answer]
do you have any code or more information you could share?

**Reply by matheuskousen** [reply]
yeah, wait a sec

**Comment by matheuskousen** [comment]
http://stylesheetdev.com here is a exemplo

**Comment by jamesmosier** [comment]
Have you tried adding the `muted` property as well?

I came across [this stackoverflow answer](https://stackoverflow.com/a/52542831/1554860) that mentions you must have `autoPlay` and `muted`.
